src/cascade/saver/save_model_results.py

```diff
@@ -0,0 +1,92 @@
+"""
+Saves results from a model fit to a database which the EpiViz can access.
+"""
+import logging
+
+import save_results._save_results as sr
+
+from cascade.saver.generate_draws import generate_draws_table
+
+from cascade.saver import DRAWS_INPUT_FILE_PATTERN, MODEL_TITLE
+
+
+CODELOG = logging.getLogger(__name__)
+MATHLOG = logging.getLogger(__name__)
+
+
+def save_model_results(execution_context):
+    """
+    We want to visualize model results using EpiViz.  To do this, we need to
+    generate draws from a model fit, and save those draws in a database
+    that EpiViz can access.
+
+    Args:
+        execution_context (ExecutionContext): contains model id data
+
+    Returns:
+        (str) of the mvid returned by save_results
+
+    """
+
+    dm_file = execution_context.dismodfile
+
+    if not dm_file:
+        raise ValueError("Must provide a dismod file")
+
+    draws_df = generate_draws_table(dm_file)
+
+    CODELOG.debug(draws_df.columns)
+    CODELOG.debug(draws_df.iloc[0])
+
+    with execution_context.scratch_dir() as scratch_dir:
```

Comment:
Using this as a context manager is perfect. But you should try to make it so only code that needs the scratch_dir is inside the manager, move everything else above or below. That way it's easy to see exactly what the context manager effects.